.gitignore

```diff
@@ -10,13 +10,16 @@
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
+# Goland
+.idea/
 ### Go Patch ###
 /vendor/
 
-# Goland
+# IdeaIDE
 .idea
 
-# VSCode
+# VS Code
 .vscode
 
-go-pre.db
\ No newline at end of file
+# Sublime
+*.sublime*
```

Comment:
why did u take off `go-pre.db`, we shouldn't commit the database

Reply:
I didn't, might have had an old version. I can add it back